packages/actor-scraper/web-scraper/README.md

```diff
@@ -10,7 +10,7 @@ you might prefer to start with [**Web Scraping 101**](https://docs.apify.com/web
 and then continue with [**Scraping with Web Scraper**](https://docs.apify.com/tutorials/apify-scrapers/web-scraper),
 a tutorial which will walk you through all the steps and provide a number of examples. Or you can just watch this video tutorial:
 
-[![Watch Web Scraper video](https://img.youtube.com/vi/K76Hib0cY0k/0.jpg)](https://youtu.be/K76Hib0cY0k)
+https://www.youtube.com/watch?v=5kcaHAuGxmY
```

Comment:
i guess we want to keep the [preview image](https://img.youtube.com/vi/5kcaHAuGxmY/0.jpg), right?

```suggestion
[![Watch Web Scraper video](https://img.youtube.com/vi/5kcaHAuGxmY/0.jpg)](https://youtu.be/5kcaHAuGxmY)
```

Reply:
The link will turn into a YouTube embed, so the image will be there alright. We just had to do workarounds before.